addons/beehave/nodes/decorators/time_limiter.gd

```diff
@@ -28,20 +36,22 @@ func tick(actor: Node, blackboard: Blackboard) -> int:
 			running_child = child
 			if child is ActionLeaf:
 				blackboard.set_value("running_action", child, str(actor.get_instance_id()))
-		
+		else:
+			child.after_run(actor, blackboard)
 		return response
 	else:
-		child.after_run(actor, blackboard)
 		interrupt(actor, blackboard)
+		child.after_run(actor, blackboard)
 		return FAILURE
 
 
-func before_run(actor: Node, blackboard: Blackboard) -> void:
```

Comment:
not required, we will do it once when time left == 0.0 (see line 22)

Reply:
This is (possibly) a breaking change, because originally the timer was reset when the limiter received `before_run` callback but with the current changes, it is never reset. This resetting behaviour is something I'm currently relying on in my behaviour tree.

Reply:
Good point. Let me ensure this behaviour is consistent - I'll revert this.